.github/workflows/bundle-app.yml

```diff
@@ -0,0 +1,71 @@
+name: bundle app
+on: [push, pull_request]
+
+jobs:
+  bundle-apk:
+    runs-on: macos-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.4.0
+        with:
+          access_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6.x
+      - name: install bundle
+        run: |
+          gem install bundler:2.1.4
+      - uses: ledgerhq/actions/check-member@v2.0.0
+        with:
+          username: ${{ github.actor }}
+          ban: ledgerlive
+        if: github.event_name == 'pull_request'
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@main
+        with:
+          node-version: 12.x
+      - name: set git user
+        run: |
+          git config user.email "team-live@ledger.fr"
+          git config user.name "Team Live"
+      - name: install dependencies
+        run: yarn --frozen-lockfile
+      - uses: ledgerhq/actions/get-package-infos@v1.0.0
+        id: version
+      - name: set beta name
+        uses: ./.github/actions/change-product-name
+      - name: Get short SHA
+        id: slug
+        run: echo "::set-output name=sha8::${GITHUB_SHA:0:7}"
+      - name: make local version
+        run: yarn version --new-version=${{ steps.version.outputs.clean }}-sha.${{ steps.slug.outputs.sha8 }}
+      - name: build the app
+        run: |
+          export ANDROID_KEYSTORE_FILE=$(pwd)/android/app/staging.kstr
+          export ANDROID_KEYSTORE_PASS=staging
+          export ANDROID_KEY_ALIAS=staging
+          export ANDROID_KEY_PASS=staging
+          bundle exec fastlane android apk --env production
```

Comment:
I'm not sure `--env production` flag is used inside fastlane.
We can replace this line with `yarn android:apk` alias

Reply:
let's try